app/models/manageiq/providers/redhat/infra_manager/event_catcher/runner.rb

```diff
@@ -41,7 +41,8 @@ def monitor_events
 
   def queue_event(event)
     _log.info "#{log_prefix} Caught event [#{event[:name]}]"
-    EmsEvent.add_queue('add_rhevm', @cfg[:ems_id], event.to_hash)
+    event_hash = ManageIQ::Providers::Redhat::InfraManager::EventParser.event_to_hash(event, @cfg[:ems_id])
+    EmsEvent.add_queue('add', @cfg[:ems_id], event_hash)
```

Comment:
So once I went to a release beyond EUWE stable, RHEV did not seem to do a refresh to validate VM. If I provisioned VMs from Admin or Self-Service they would get stuck on "Validating VM". If I did a manual refresh or waited a few hours for a refresh to happen on its own it kicked past the "validating vms" and completed.